src/Cli/dotnet/Program.cs

```diff
@@ -29,55 +30,91 @@ public static int Main(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
-            new MulticoreJitActivator().TryActivateMulticoreJit();
+            // Capture the current timestamp to calculate the host overhead.
+            DateTime mainTimeStamp = DateTime.Now;
 
-            if (Env.GetEnvironmentVariableAsBool("DOTNET_CLI_CAPTURE_TIMING", false))
+            // Determine if the perf log is enabled.
```

Comment:
nit: remove unnecessary comment. "Determine if the perf log is enabled.", you can tell by the variable name. Program.cs already has a lot of clutter.

Reply:
Removed